```
[ERROR] [checkstyle] [ERROR] /home/vsts/work/1/s/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example1.java:14:10: 'enum def ident' has incorrect indentation level 4, expected level should be 2. [Indentation]
[ERROR] [checkstyle] [ERROR] /home/vsts/work/1/s/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example1.java:15:5: 'enum def rcurly' has incorrect indentation level 4, expected level should be 2. [Indentation]
[ERROR] [checkstyle] [ERROR] /home/vsts/work/1/s/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example1.java:19:5: 'method def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]
[ERROR] [checkstyle] [ERROR] /home/vsts/work/1/s/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example2.java:16:10: 'enum def ident' has incorrect indentation level 4, expected level should be 2. [Indentation]
[ERROR] [checkstyle] [ERROR] /home/vsts/work/1/s/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example2.java:17:5: 'enum def rcurly' has incorrect indentation level 4, expected level should be 2. [Indentation]
[ERROR] [checkstyle] [ERROR] /home/vsts/work/1/s/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example2.java:21:5: 'method def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]

```

--- a/src/site/xdoc/checks/sizes/outertypenumber.xml
+++ b/src/site/xdoc/checks/sizes/outertypenumber.xml
@@ -52,16 +52,15 @@
 </code></pre></div>
         <p id="Example1-code">Example:</p>
         <div class="wrapper"><pre class="prettyprint"><code class="language-java">
-// violation 2 lines above 'Outer types defined is 3 (max allowed is 1)'
-
+package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
+// violation above 'Outer types defined is 2 (max allowed is 1)'
 public class Example1 {
+  enum outer {
+  }
 }
 
-class Example {
+class ExampleOne {
   void exampleMethod() {}
-}
-
-enum outer {
 }
 </code></pre></div><hr class="example-separator"/>
 
@@ -79,14 +78,15 @@ enum outer {
 </code></pre></div>
         <p id="Example2-code">Example:</p>
         <div class="wrapper"><pre class="prettyprint"><code class="language-java">
+package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
+
 public class Example2 {
+  enum outer {
+  }
 }
 
-class Examples {
+class ExampleTwo {
   void exampleMethod() {}
-}
-
-enum outer1 {
 }
 </code></pre></div>
 

--- a/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsExamplesAstConsistencyTest.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsExamplesAstConsistencyTest.java
@@ -111,7 +111,6 @@ public class XdocsExamplesAstConsistencyTest {
             "checks/metrics/classdataabstractioncoupling/ignore/Example9",
             "checks/metrics/classdataabstractioncoupling/ignore/deeper/Example5",
             "checks/metrics/classdataabstractioncoupling/ignore/deeper/Example6",
-            "checks/sizes/outertypenumber/Example2",
             "checks/whitespace/parenpad/Example2",
             // Note: customImport/ImportOrder changes import group ORDER affecting AST structure
             "checks/imports/customimportorder/Example10",

--- a/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/sizes/OuterTypeNumberCheckExamplesTest.java
+++ b/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/sizes/OuterTypeNumberCheckExamplesTest.java
@@ -35,7 +35,7 @@ public class OuterTypeNumberCheckExamplesTest extends AbstractExamplesModuleTest
     @Test
     public void testExample1() throws Exception {
         final String[] expected = {
-            "9:1: " + getCheckMessage(MSG_KEY, 3, 1),
+            "10:1: " + getCheckMessage(MSG_KEY, 2, 1),
         };
 
         verifyWithInlineConfigParser(getPath("Example1.java"), expected);

--- a/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example1.java
+++ b/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example1.java
@@ -6,17 +6,15 @@
 </module>
 */
 
-package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
 // xdoc section -- start
-// violation 2 lines above 'Outer types defined is 3 (max allowed is 1)'
-
+package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
+// violation above 'Outer types defined is 2 (max allowed is 1)'
 public class Example1 {
+  enum outer {
+  }
 }
 
-class Example {
+class ExampleOne {
   void exampleMethod() {}
-}
-
-enum outer {
 }
 // xdoc section -- end

--- a/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example2.java
+++ b/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/Example2.java
@@ -8,15 +8,15 @@
 </module>
 */
 
-package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
 // xdoc section -- start
+package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
+
 public class Example2 {
+  enum outer {
+  }
 }
 
-class Examples {
+class ExampleTwo {
   void exampleMethod() {}
-}
-
-enum outer1 {
 }
 // xdoc section -- end

